lib/Utils.hs

```diff
@@ -44,11 +44,11 @@ ekbind k f cb = k $ either (cb . Left) (($ cb) . f)
 eklift :: K i o a -> EK i o e a
 eklift = kmap Right
 
-kcodensity :: Monad m => m a -> (forall r. K (m r) (m r) a)
+kcodensity :: Monad m => m a -> K (m r) (m r) a
 kcodensity = (>>=)
 
 kuncodensity :: Monad m => (forall r. K (m r) (m r) a) -> m a
-kuncodensity = ($ pure)
+kuncodensity k = k pure
```

Comment:
These definitions are no longer well-typed in GHC 9.2 because of simplified subsumption. Patched versions are in a sense "the same thing" but pass GHC's new checker.